src/main/java/com/gaggle/contact/domain/Contact.java

```diff
@@ -14,11 +14,13 @@
 
     private @Id @GeneratedValue Long id;
     private @NotBlank String name;
+    private String email;
 
     public Contact() {}
 
-    public Contact(final String name) {
```

Comment:
The email is not indicated as required so the constructors should have been overloaded so you can have
public Contact(String name) and
public Contact(String name, String email){
  super(name);
  this.email = email;
...